src/rtc/h5_prep.py

```diff
@@ -300,20 +309,196 @@ def get_metadata_dict(product_id: str,
+
+        # file format spec. for local inc. angle - for 2.4
+        'RTC/metadata/fileFormatSpecification/localIncidenceAngle/sampleType':  # 2.4
+            ['local_inc_sample_type', 'Angle', 'Sample type of the data'],
+        'RTC/metadata/fileFormatSpecification/localIncidenceAngle/dataFormat':
+            ['local_inc_data_format', cfg_in.groups.product_group.output_imagery_format,
+             'data format of the data'],
+        'RTC/metadata/fileFormatSpecification/localIncidenceAngle/dataType':
+            ['local_inc_data_type', 'Float',
+             'data type of the data'],
+        'RTC/metadata/fileFormatSpecification/localIncidenceAngle/bitsPerSample':
+            ['local_inc_bits_per_sample', 32,
+             'Bytes per pixel'],
+        'RTC/metadata/fileFormatSpecification/localIncidenceAngle/byteOrder':
+            ['local_inc_byte_order', sys.byteorder,
+             'Byte order of the data'],
+
+        # file format spec. for backscatter - for 3.1
+        'RTC/metadata/fileFormatSpecification/backScatter/measurementType':  # 3.1
+            ['measurement_type', 'Gamma0', 'Sample type of the data'],
+        'RTC/metadata/fileFormatSpecification/backScatter/expressionConvention':  # 3.1
+            ['measurement_expression_convention', 'Linear amplitude', #TODO Confirm
+             'Backscatter expression convention'],
+        # NOTE: polarization is provided in 'listOfPolarizations'
+        'RTC/metadata/fileFormatSpecification/backScatter/dataFormat':
+            ['measurement_data_format', cfg_in.groups.product_group.output_imagery_format,
+             'data format of the measurement'],
+        'RTC/metadata/fileFormatSpecification/backScatter/dataType':
+            ['measurement_data_type', 'Float',
+             'data type of the measurement'],
+        'RTC/metadata/fileFormatSpecification/backScatter/bitsPerSample':
+            ['measurement_bits_per_sample', 32,
+             'Bytes per sample in the measurement'],
+        'RTC/metadata/fileFormatSpecification/backScatter/byteOrder':
+            ['measurement_byte_order', sys.byteorder,
+             'Byte order of the measurement'],
+
+        # The field below to be populated only when using power in dB scale
+        #'RTC/metadata/fileFormatSpecification/scalingConversion': # 3.2
+        #    ['',
+        #     ('equation to convert from pixel linear amplitude / power to'
+        #      'logarithmic decibel scale')],
+
+        'RTC/metadata/processingInformation/noiseRemovalApplied':  # 3.3
+            ['noise_removal_applied',
+             cfg_in.groups.processing.apply_thermal_noise_correction,
+             'A flag to indicate whether noise removal was applied'],
+    
+        'RTC/metadata/processingInformation/geoidReference':  # for 4.2
+            ['geoid_source_description', 'EGM2008', 'Geoid source description'], #TODO confirm, might need to be populated via runconfig
```

Comment:
```suggestion
    
        #'RTC/metadata/processingInformation/geoidReference':  # for 4.2
        #    ['geoid_source_description', 'EGM2008', 'Geoid source description'], #TODO confirm, might need to be populated via runconfig
```
Let's comment this for now